config/locales/devise.en.yml

```
@@ -0,0 +1,63 @@
+      unlock_instructions:
+        subject: "Unlock instructions"
+      email_changed:
+        subject: "Email Changed"
+      password_change:
+        subject: "Password Changed"
+    omniauth_callbacks:
+      failure: "Could not authenticate you from %{kind} because \"%{reason}\"."
+      success: "Successfully authenticated from %{kind} account."
+    passwords:
+      no_token: "You can't access this page without coming from a password reset email. If you do come from a password reset email, please make sure you used the full URL provided."
+      send_instructions: "You will receive an email with instructions on how to reset your password in a few minutes."
+      send_paranoid_instructions: "If your email address exists in our database, you will receive a password recovery link at your email address in a few minutes."
+      updated: "Your password has been changed successfully. You are now signed in."
+      updated_not_active: "Your password has been changed successfully."
+    registrations:
+      destroyed: "Bye! Your account has been successfully cancelled. We hope to see you again soon."
+      signed_up: "Welcome! You have signed up successfully."
+      signed_up_but_inactive: "You have signed up successfully. However, we could not sign you in because your account is not yet activated."
+      signed_up_but_locked: "You have signed up successfully. However, we could not sign you in because your account is locked."
+      signed_up_but_unconfirmed: "A message with a confirmation link has been sent to your email address. Please follow the link to activate your account."
+      update_needs_confirmation: "You updated your account successfully, but we need to verify your new email address. Please check your email and follow the confirmation link to confirm your new email address."
+      updated: "Your account has been updated successfully."
+      updated_but_not_signed_in: "Your account has been updated successfully, but since your password was changed, you need to sign in again."
+    sessions:
+      signed_in: "Signed in successfully."
+      signed_out: "Signed out successfully."
+      already_signed_out: "Signed out successfully."
+    unlocks:
+      send_instructions: "You will receive an email with instructions for how to unlock your account in a few minutes."
+      send_paranoid_instructions: "If your account exists, you will receive an email with instructions for how to unlock it in a few minutes."
+      unlocked: "Your account has been unlocked successfully. Please sign in to continue."
+  errors:
+    messages:
+      already_confirmed: "was already confirmed, please try signing in"
+      confirmation_period_expired: "needs to be confirmed within %{period}, please request a new one"
+      expired: "has expired, please request a new one"
+      not_found: "not found"
+      not_locked: "was not locked"
+      not_saved: "Your %{resource} could not be saved because of the following problems:"
```

```suggestion
  errors:
    messages:
      already_confirmed: "already confirmed, try signing in"
      confirmation_period_expired: "must be confirmed within %{period}, request a new one"
      expired: "expired, request a new one"
      not_found: "not found"
      not_locked: "not locked"
      not_saved: "%{resource} could not be saved because of the following problems:"
```
...is there a way to upcase resource?

---

I wasn't able to find a way to do that with i18n in https://guides.rubyonrails.org/i18n.html#organization-of-locale-files etc, so I put the `Your` before it to help out with that

---

```suggestion
  errors:
    messages:
      already_confirmed: "already confirmed, try signing in"
      confirmation_period_expired: "must be confirmed within %{period}, request a new one"
      expired: "expired, request a new one"
      not_found: "not found"
      not_locked: "not locked"
      not_saved: "Your %{resource} could not be saved because of the following problems:"
```
All the others look good though!

---

Alas. Yeah that seems good then.